odata-url-conventions/5 Query Options.md

```diff
@@ -2001,7 +2004,7 @@ http://host/service/Categories?$expand=Products/Sales.PremierProduct/$ref($filte
 ```
 :::
 
-Cyclic navigation properties (whose target type is identical or can be
+<a name="ExpandOptionlevels">Cyclic navigation properties (whose target type is identical or can be
```

Comment:
```suggestion
<a name="ExpandOptionlevels">Cyclic navigation properties</a> (whose target type is identical or can be
```
Otherwise the named anchor refers to the entire paragraph (and the entire paragraph is highlighted when visiting the anchor).

Reply:
Which is what I wanted and like

Reply:
Alternatively we could restrict that to the first sentence

Reply:
I leave that up to you.